trade-finance-manager-ui/server/controllers/case/index.js

```diff
@@ -318,7 +318,7 @@ const getCaseFacility = async (req, res) => {
     allAmendments,
     amendments,
     amendmentsInProgress,
-    isTfmFacilityEndDateFeatureFlagEnabled: isTfmFacilityEndDateFeatureFlagEnabled(),
+    showFacilityEndDate: isTfmFacilityEndDateFeatureFlagEnabled() && facility.facilitySnapshot.isGef,
```

Comment:
Can we update the unit test please, there's only 1x test for this with a `false` boolean

Perhaps we could simply replace that with `isTfmFacilityEndDateFeatureFlagEnabled() && facility.facilitySnapshot.isGef`

Reply:
added tests